packages/components/workbench/src/WorkbenchHeader/WorkbenchHeader.tsx

```diff
@@ -10,7 +11,7 @@ import { getWorkbenchHeaderStyles } from './WorkbenchHeader.styles';
 
 export interface WorkbenchHeaderProps extends CommonProps {
   /** This is the title that will be shown inside the Header component */
-  title: string;
+  title: ReactNode;
```

Comment:
Maybe you could include the change in line 61 in this PR

Basically, if title is a string we render the way we render it now, but if is a ReactNode we just render the ReactNode

wdyt?

Reply:
@denkristoffer any chance of following up with this PR changes?